src/api/index.js

```diff
@@ -26,3 +26,41 @@ export const login = async (username, password) => {
     console.error(err);
   }
 };
+
+export const getProducts = async () => {
+  try {
+    const response = await fetch(`${BASE_URL}/products`, {
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    });
+    const results = await response.json();
+    return results;
+
+  } catch (err) {
+    console.log('Error getting products')
+  }
+};
+
+export const updateProduct = async ({token, name, description, price, thumbnailImage, productId}) => {
```

Comment:
`token` parameter should be outside the object
You can also just have the 2nd parameter be `product` and pass that directly in the body of the JSON request, no destructuring needed